src/nikita/Indices.java

```diff
@@ -0,0 +1,31 @@
+package nikita;
+
+public class Indices {
+
+
+	int[] sum(int[] n, int targetnum) {
+		
+		for(int i=0;i<n.length;i++) {
+			for(int j=0;j<n.length;j++) {
```

Comment:
Inner for loop should from i+1, think on this, if you want to connect on Skypethen feel free to message.

Take reference of this code
Kangan - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/987/files
Rupali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/991/files
Dipali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/1003/files